### ISSUE TITLE
Too few arguments to function Baldinof\RoadRunnerBundle\Worker\Dependencies::__construct()

### ISSUE THREAD

**Issue by aldump** [Baldinof/roadrunner-bundle#26]
After updating to version 1.5.0 start receiving following exception

Too few arguments to function Baldinof\RoadRunnerBundle\Worker\Dependencies::__construct(), 2 passed in /var/www/var/cache/prod/ContainerXu7JtGX/getDependenciesService.php on line 20 and exactly 3 expected  

All because in config file its still configuration for 2 arguments

```
    $services->set(Dependencies::class)
        ->public() // Manually retrieved on the DIC in the Worker if the kernel has been rebooted
        ->args([
            service(MiddlewareStack::class),
            service(KernelRebootStrategyInterface::class),
        ]);
```

**Comment by aldump** [comment]
Here is [PR](https://github.com/Baldinof/roadrunner-bundle/pull/27) to fix it

**Comment by Baldinof** [comment]
Thank you! The PR is merged and released under 1.5.1

**Comment by aldump** [comment]
Fixed! Thanks for quick reaction!
